### PR TITLE
chore: added pre-release tag for workflow publications

### PR DESCRIPTION
**Pull request by henit-chobisa**
## Tasks
- [x] Added `pre-release` tag for running release workflow on `releases` as well as `pre-releases`

### CODE REVIEW

**Review by pablohashescobar — CHANGES_REQUESTED** [review]

File: docker-compose-hub.yml [hunk 1]
```diff
@@ -75,6 +75,8 @@ services:
                 image: makeplane/plane-backend:latest
                 restart: always
                 command: ./bin/takeoff
+                ports:
```
[the remark this answers is not in the document]
Comment (henit-chobisa) [reply docker-compose-hub.yml]: Removed it ✅

**Review by pablohashescobar — APPROVED** [review]

**Comment by pablohashescobar** [comment]
@henit-chobisa just to confirm we will be removing the worker images from now on right ?

**Comment by henit-chobisa** [comment]
> @henit-chobisa just to confirm we will be removing the worker images from now on right ? Everything will be using backend container only ?

Yes @pablohashescobar Nikhil, the backend container is capable for worker and beat as well, just the change of command. Hence, we can use the same backend image for the worker purposes as well.

**Comment by pablohashescobar** [comment]
Awesome. Let's update the docs as well.

**Comment by henit-chobisa** [comment]
@pablohashescobar 
Sure Nikhil, also just to clarify, the compose hub won't work until a user exposes the backend port, I have to make the change to localhost thing.

**Comment by pablohashescobar** [comment]
Yep let's make it localhost.

**Review by sriramveeraghanta — APPROVED** [review]

**Pull request merged**